### PR TITLE
Updated MPF upgrade tutorial/example

### PR DESCRIPTION
**Pull request by nickanderson**
This document was authored in org-mode as a literate document. It was exported
to markdown

### CODE REVIEW

**Comment by nickanderson** [comment]
@cf-bottom jenkins, please

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/fast-build-and-deploy-docs-3.18/88/)

**Jenkins:** https://ci.cfengine.com/job/fast-build-and-deploy-docs-3.18/88/

**Documentation:** http://buildcache.cfengine.com/packages/build-documentation-pr/jenkins-fast-build-and-deploy-docs-3.18-88/output/_site/

**Comment by nickanderson** [comment]
The link to this page in the preview build: http://buildcache.cfengine.com/packages/build-documentation-pr/jenkins-fast-build-and-deploy-docs-3.18-88/output/_site/examples-tutorials-masterfiles_policy_framework_upgrade.html

File: examples/tutorials/masterfiles_policy_framework_upgrade.markdown [hunk 1]
```diff
@@ -6,163 +6,130 @@ sorting: 14
...
+
+
+## Validating expectations
+
+From `/tmp/MPF-upgrade/integration/masterfiles`. Let's inspect what we expect.
+
+Is it the root of a policy set? `promises.cf` will be present if so.
+
+```bash
+cd $INTEGRATION_ROOT/masterfiles
```
Comment (craigcomstock) [inline examples/tutorials/masterfiles_policy_framework_upgrade.markdown:26]:
```suggestion
export INTEGRATION_ROOT=/tmp/MPF-upgrade/integration
cd $INTEGRATION_ROOT/masterfiles
```

File: examples/tutorials/masterfiles_policy_framework_upgrade.markdown [hunk 2]
```diff
@@ -171,724 +138,798 @@ Changes not staged for commit:
...
-[root@hub masterfiles]# ./autogen.sh
-configure.ac:31: installing `./config.guess'
-configure.ac:31: installing `./config.sub'
-configure.ac:34: installing `./install-sh'
-configure.ac:34: installing `./missing'
-checking build system type... x86_64-unknown-linux-gnu
-checking host system type... x86_64-unknown-linux-gnu
-checking target system type... x86_64-unknown-linux-gnu
+make
+make install prefix=$INTEGRATION_ROOT/
```
Comment (craigcomstock) [inline examples/tutorials/masterfiles_policy_framework_upgrade.markdown:261]: DESTDIR here?

**Review by craigcomstock — APPROVED** [review]
looks good here! will check out the rendered version as well

File: examples/tutorials/masterfiles_policy_framework_upgrade.markdown (hunk 2 above)
Comment (nickanderson) [reply examples/tutorials/masterfiles_policy_framework_upgrade.markdown:286]: I know, I oped against it becasue it makes a deep directory, and for someone who is probably cloning the root of their policy to a place it's a bit weird to me to have them be sure to clone it under var/cfengine

File: examples/tutorials/masterfiles_policy_framework_upgrade.markdown (hunk 1 above)
Comment (nickanderson) [reply examples/tutorials/masterfiles_policy_framework_upgrade.markdown]: I think i cleaned that up

**Comment by nickanderson** [comment]
@cf-bottom jenkins please

File: examples/tutorials/masterfiles_policy_framework_upgrade.markdown (hunk 2 above)
Comment (craigcomstock) [reply examples/tutorials/masterfiles_policy_framework_upgrade.markdown:286]: DESTDIR makes a deeper copy with many subdirs like `var/cfengine` so awkward for this procedure.

**Comment by nickanderson** [comment]
Um @cf-bottom are you awake? Jenkins pretty please

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/fast-build-and-deploy-docs-3.18/89/)

**Jenkins:** https://ci.cfengine.com/job/fast-build-and-deploy-docs-3.18/89/

**Documentation:** http://buildcache.cfengine.com/packages/build-documentation-pr/jenkins-fast-build-and-deploy-docs-3.18-89/output/_site/

**Pull request merged**